internal/report/report.go

```diff
@@ -26,23 +26,20 @@ import (
 )
 
 // ExternalServiceURL is the base HTTP URL for a telemetry service.
-// If not otherwise specified it will use the hard coded default.
+// If not otherwise specified, it will use the hard-coded default.
 //
 // Override at build time via:
 //
 //	-ldflags "-X github.com/open-policy-agent/opa/internal/report.ExternalServiceURL=<url>"
 //
 // This will be overridden if the OPA_TELEMETRY_SERVICE_URL environment variable
 // is provided.
-var ExternalServiceURL = "https://telemetry.openpolicyagent.org"
+var ExternalServiceURL = "https://api.github.com"
 
 // Reporter reports information such as the version, heap usage about the running OPA instance to an external service
-type Reporter struct {
-	body   map[string]any
-	client rest.Client
-
-	gatherers    map[string]Gatherer
-	gatherersMtx sync.Mutex
+type Reporter interface {
```

Comment:
Making `Reporter` an interface might be overkill as it's pretty unlikely someone relies on this, but this way it's possible to swap in a bespoke reporter for posting meta.

The type change is however a breaking change to `report.New()`, but as said above: very unlikely someone is using this today.